lib/gitlab/client/build_triggers.rb

```diff
@@ -21,7 +22,7 @@ def triggers(project)
     # @param  [Integer] project The ID of a project.
     # @param  [String] token The token of a trigger.
     # @return [Gitlab::ObjectifiedHash] The trigger.
-    def trigger(project, token)
+    def trigger_detail(project, token)
```

Comment:
The difference here is that `trigger` could be interpreted as either a verb or a noun, it was this ambiguity I was attempting to avoid.
Do you have any views on the naming of the counterpart `trigger_build` method?